pkg/agent/apiserver/handlers/podinterface/handler.go

```diff
@@ -72,3 +74,14 @@ func HandleFunc(aq monitor.AgentQuerier) http.HandlerFunc {
 		}
 	}
 }
+
+var _ common.TableOutput = new(Response)
+
+func (r Response) GetTableHeader() []string {
+	return []string{"NAMESPACE", "NAME", "INTERFACE-NAME", "IP", "MAC", "PORTUUID", "OFPORT", "CONTAINER"}
+}
+
+func (r Response) GetTableRow(maxColumnLength int) []string {
+	row := []string{r.PodNamespace, r.PodName, r.InterfaceName, r.IP, r.MAC, r.PortUUID, strconv.Itoa(int(r.OFPort)), r.ContainerID}
```

Comment:
Container ID is too long, we can show the first 12 chars like `docker ps` does.

Reply:
it could be helpful to add aux functions for the Response struct that return a stringified ofport and the short name for the container_id

Reply:
I agree with the suggestions and added aux functions. It helps to make the code more tidy.